am-lib/src/main/java/uk/gov/hmcts/reform/amlib/models/ExplicitAccessGrant.java

```diff
@@ -6,14 +6,15 @@
 import uk.gov.hmcts.reform.amlib.enums.Permission;
 import uk.gov.hmcts.reform.amlib.enums.SecurityClassification;
 
+import java.util.List;
 import java.util.Map;
 import java.util.Set;
 
 @Data
 @Builder
 public final class ExplicitAccessGrant {
     private final String resourceId;
-    private final String accessorId;
+    private final List<String> accessorId;
```

Comment:
Please rename since you turned singular field into list .

Reply:
Shouldn't that be a set?

Reply:
👍